### PR TITLE
feat: Sets jre11 as minimum required.

### PR DESCRIPTION
**Pull request by damencho**
Allows running it with java17. Drops unused dependency.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -9,7 +9,7 @@ Homepage: https://jitsi.org
 
 Package: jigasi
 Architecture: any
-Depends: ${misc:Depends}, openjdk-11-jre-headless | openjdk-11-jre | java11-runtime-headless | java11-runtime, libxss1, openssl, ruby-hocon
+Depends: ${misc:Depends}, default-jre-headless (>= 2:1.11), openssl, ruby-hocon
```
Comment (ibauersachs) [inline debian/control:12]: What's with the ruby-hocon thing?

Comment (damencho) [reply debian/control:12]: https://github.com/jitsi/jigasi/blob/296b7cd94c0712d586737b1e59dd8d12f453a475/debian/postinst#L163

Comment (damencho) [reply debian/control:12]: And of course the:
https://github.com/jitsi/jigasi/blob/296b7cd94c0712d586737b1e59dd8d12f453a475/debian/postinst#L164

Comment (ibauersachs) [reply debian/control:12]: Urgh.

**Review by ibauersachs — APPROVED** [review]

**Pull request merged**